x-pack/plugins/index_lifecycle_management/__jest__/client_integration/edit_policy/features/frozen_phase.test.ts

```diff
@@ -28,28 +29,19 @@ describe('<EditPolicy /> frozen phase', () => {
     httpRequestsMockHelpers.setDefaultResponses();
 
     await act(async () => {
-      testBed = await setup();
+      testBed = await initTestBed();
```

Comment:
I love the readability of factory functions that state what they return, and then assigning the return value to a variable with the same name! It makes the code so much more transparent for me. Thanks for making this kind of change everywhere.